<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatroomsChooser</class>
 <widget class="QDialog" name="ChatroomsChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chatrooms</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter the chatrooms to watch into the following field. Separate them by comma.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="roomnamesField"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="saveButton">
         <property name="toolTip">
          <string>Apply setting and close</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/Icons/res/apply.svg</normaloff>:/Icons/res/apply.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="defaultButton">
         <property name="text">
          <string>Restore Defaults</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/Icons/res/default.svg</normaloff>:/Icons/res/default.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
